modules/ob-installing-buildah-strategy.adoc

```diff
@@ -0,0 +1,22 @@
+// This module is included in the following assembly:
+//
+// builds/installing-sample-build-strategies.adoc
+
+:_content-type: PROCEDURE
+[id="installing-buildah-strategy_{context}"]
+= Installing buildah strategy
+
+You can install the `buildah` cluster build strategy, which does not require a daemon to run and is available for use by unprivileged users.
+
+.Prerequisites
+
+* You have installed the `oc` CLI.
+
+.Procedure
+
+* To install the `buildah` strategy, run the following command :
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/shipwright-io/build/main/samples/buildstrategy/buildah/buildstrategy_buildah_shipwright_managed_push_cr.yaml
```

Comment:
```suggestion
$ oc apply -f https://raw.githubusercontent.com/shipwright-io/build/main/samples/buildstrategy/buildah/buildstrategy_buildah_strategy_managed_push_cr.yaml
$ oc apply -f https://raw.githubusercontent.com/shipwright-io/build/main/samples/buildstrategy/buildah/buildstrategy_buildah_shipwright_managed_push_cr.yaml
```

We need to have both the commands here.

Reply:
@apoorvajagtap need a clarity: Does a user need to run both the cmds or one of the commands to get the buildah startegy installed?

Reply:
@shipsing both strategies are independent, so, a user can run either both the commands or just one command (it's up to their requirement/understanding), having both the strategies in the cluster won't affect anything. We are inclined to highlight both commands here so that users are aware of the strategies we provide.